### ISSUE TITLE
[quasar v0.17.23]  [google chrome v73], mouse scroll event warning

### ISSUE THREAD

**Issue by Lin84** [quasarframework/quasar#3696]
Context:
1.) quasar-cli": "^0.17.23",
2.) chrome: Version 73.0.3683.86 (Official Build) (64-bit)
3.) mouse scroll event

Problem:
after chrome was updated to latest version [73.0.3683.86]
if you open the modal and scroll
in console log you will see warning:
"[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive."

Demo:
https://monosnap.com/file/IvPUgeS9QgDOpOzYJmfXdQNJXLLplv

Additional information:
the warning is trigger by calling function: "stopAndPrevent"
![image](https://user-images.githubusercontent.com/7355572/55010796-26e29780-4fe5-11e9-8063-b48fbd00af40.png)

**Comment by borie88** [comment]
Same issue here when dialog is open and I scroll

**Comment by Jag-Nahl** [comment]
Also getting the same error here

**Comment by Lin84** [comment]
the fix can be done by adding following code, please test it if everything is ok:
![image](https://user-images.githubusercontent.com/7355572/56093359-22185180-5ec8-11e9-8c97-403fe35e386d.png)

**Comment by Lin84** [comment]
quick fix for quasar v17:
https://github.com/quasarframework/quasar/pull/3889

please test it if everything is ok

**Comment by jlanzarotti** [comment]
I think the solution goes by patching the stopAndPrevent function in events.js. Adding passive there seems not correct.

``` js
export function stopAndPrevent (e) {
  e.cancelable !== false && e.preventDefault()
  e.stopPropagation();
}
```

This is actually how it is in 1.0 anyway.

**Comment by Lin84** [comment]
create brand new pull request with updated solution as @jlanzarotti mentioned in last commet

**Comment by rstoenescu** [comment]
Hi,

We're currently under heavy load with v1, which I recommend to upgrade to as soon as possible.
If your business requires this fix in 0.17, let me know. But just to paint the bigger picture: fixing this for 0.17 (merging PR and testing all use cases) requires a whole great deal of effort, so please make sure that you really really need this otherwise it is very detrimental to Quasar to halt working on v1.

Thanks.

**Comment by Lin84** [comment]
no problem, thx for your response :)

**Comment by pdmshrestha** [comment]
@rstoenescu  I am working on legacy project using 0.17. Currently the company has no plan/resources to upgrade to v1. Would it be possible to provide this fix for v0.17?
